core/src/main/java/org/apache/accumulo/core/client/summary/CounterSummary.java

```diff
@@ -54,7 +54,8 @@ public CounterSummary(Summary summary, boolean checkType) {
     if (checkType) {
       String className = summary.getSummarizerConfiguration().getClassName();
       try {
-        getClass().getClassLoader().loadClass(className).asSubclass(CountingSummarizer.class);
+        var unusedRetVal =
+            getClass().getClassLoader().loadClass(className).asSubclass(CountingSummarizer.class);
```

Comment:
This might make errorprone happy, but it adds compiler warnings about unused variables, at least in Eclipse (maybe also with javac?). One option is to actually use it. Could add a trace log message that uses it after it is assigned.

Reply:
Added log.trace message to make use of unused return value from loadclass method.